fitbert_test_data/get_options.py

```diff
@@ -0,0 +1,425 @@
+                "will",
+                "",
+                VB,
+                "future simple",
+                "future",
+                "active",
+                ["1S", "2S", "3S", "1P", "2P", "3P"],
+            ]
+        )  # will write
+        verb_forms.append(
+            [
+                "will",
+                "be",
+                VBN,
+                "future simple",
+                "future",
+                "passive",
+                ["1S", "2S", "3S", "1P", "2P", "3P"],
+            ]
+        )  # will be written
+        verb_forms.append(
+            [
+                "will",
+                "be",
+                VBG,
+                "future continuous",
+                "future",
+                "active",
+                ["1S", "2S", "3S", "1P", "2P", "3P"],
+            ]
+        )  # will be writing
+        verb_forms.append(
+            [
+                "will",
+                "be being",
+                VBN,
+                "future continuous",
+                "future",
+                "passive",
+                ["1S", "2S", "3S", "1P", "2P", "3P"],
+            ]
+        )  # will be being written
+        verb_forms.append(
+            [
+                "will",
+                "have",
+                VBN,
+                "future perfect simple",
+                "future",
+                "active",
+                ["1S", "2S", "3S", "1P", "2P", "3P"],
+            ]
+        )  # will have written
+        verb_forms.append(
+            [
+                "will",
+                "have been",
+                VBN,
+                "future perfect simple",
+                "future",
+                "passive",
+                ["1S", "2S", "3S", "1P", "2P", "3P"],
+            ]
+        )  # will have been written
+        verb_forms.append(
+            [
+                "will",
+                "have been",
+                VBG,
+                "future perfect continuous",
+                "future",
+                "active",
+                ["1S", "2S", "3S", "1P", "2P", "3P"],
+            ]
+        )  # will have been writing
+        verb_forms.append(
+            [
+                "will",
+                "have been being",
+                VBN,
+                "future perfect continuous",
+                "future",
+                "passive",
+                ["1S", "2S", "3S", "1P", "2P", "3P"],
+            ]
+        )  # will have been being written
+
+    return verb_forms
+
+
+def get_verb_options(
+    verb, verb_phrase_inffix="", tense=["present", "past", "future"], form=["active"]
+):
+    verb_forms = conjugate(verb)
+    options = []
+    diff_order = ["there", "here", "everywhere", "elsewhere", "anywhere", "nowhere"]
+    for verb_form in verb_forms:
+        if (verb_form[4] in tense) and (verb_form[5] in form):
+            if verb_phrase_inffix.lower() in diff_order:
+                elements = [verb_phrase_inffix, verb_form[0]] + verb_form[1:3]
```

Comment:
Same with this one - can the code use functions or something to make more clear what is happening here?

Reply:
Changed to filters over dictionaries.